systemtest/playlist.xml

```diff
@@ -2161,6 +2163,7 @@
 		<groups>
 			<group>system</group>
 		</groups>
+		<platformRequirements>^arch.ppc</platformRequirements>
```

Comment:
ppcle will also be excluded. Is that expected?

Reply:
yes, need to exclude from all AIX builds.

Reply:
os.aix ?

Reply:
hmm.. that's better, changing to os.aix